packages/strapi-admin/services/user.js

```diff
@@ -66,7 +66,7 @@ const findRegistrationInfo = async registrationToken => {
  * @param {Object} params.userInfo user info
  */
 const register = async ({ registrationToken, userInfo }) => {
-  const matchingUser = await strapi.query('user', 'admin').findOne({ registrationToken });
+  const matchingUser = await strapi.query('user', 'admin').findOne({registrationToken});
```

Comment:
It's prettier that does this?

Reply:
Since I didn't touch this, I suppose it's Prettier. It's weird, though, I thought the rule was to have a space between curly braces and characters.  🤔

Reply:
I thought the same 🤔